lib/ansible/modules/database/postgresql/postgresql_facts.py

```diff
@@ -0,0 +1,921 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Andrew Klychkov (@Andersson007) <aaklychkov@mail.ru>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: postgresql_facts
+short_description: Gather facts about remote PostgreSQL servers
+description:
+- Gathers facts about remote PostgreSQL servers.
+version_added: "2.8"
+options:
+  filter:
+    description:
+    - Limit collected facts by comma separated string or YAML list.
+    - Allowable values are C(version),
+      C(databases), C(settings), C(tablespaces), C(languages), C(roles), C(namespaces),
+      C(replications), C(repl_slots), C(extensions).
+    - By default, collects all subsets.
+    - You can use shell-style (fnmatch) wildcard to pass groups of values (see Examples).
+    - You can use '!' before value (for example, C(!settings)) to exclude it from facts.
+    - If you pass including and excluding values to the filter, for example, I(filter=!settings,ver),
+      the excluding values will be ignored.
+    type: list
+  db:
```

Comment:
This should rather be login_db

Reply:
Sounds good.
In all of the existing modules they're used db, port, etc. So, I'll add aliases: login_db, login_port,...